ghostfolio/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       APP_PORT: 3334
 
   server:
-    image: ghostfolio/ghostfolio:1.287.0@sha256:4dfee326c1574c3512a5a58e7c73b9ac03535d439e2a2dab1d8cae95129e36b7
+    image: ghostfolio/ghostfolio:2.10.0@sha256:d980f3e6688c68c5c3b405f50b6539e477991916fa9b50b9384f08daa2b0de88
```

Comment:
Hey @romalane , The digest provided here is only for Amd64. Please use this digest as this is multi-architecture -> `sha256:4f02b2a2ce7c37ce76a51804172161db91d3424651db97c7a0748c41af63c460`

This will allow Ghostfolio to run on different architectures. With current changes it will crash on Arm64 machines.